src/main/java/org/team1540/robot2024/RobotContainer.java

```diff
@@ -214,17 +215,24 @@ private void configureButtonBindings() {
 
         new Trigger(indexer::isNoteStaged).and(intakeCommand::isScheduled).onTrue(CommandUtils.rumbleCommandTimed(driver.getHID(), 0.8, 0.4));
 
-        new Trigger(RobotController::getUserButton).toggleOnTrue(Commands.startEnd(
-                () -> {
-                    elevator.setBrakeMode(false);
-                    leds.setPatternAll(() -> new LedPatternRSLState(Color.kMagenta), Leds.PatternLevel.ELEVATOR_STATE);
-                },
+
+        BooleanSupplier isPreMatch = () -> (!DriverStation.isDSAttached() || DriverStation.isAutonomous()) && DriverStation.isDisabled();
```

Comment:
It would be nice if this also checked the FMS connection state, since it would be inconvenient to need to put the DS in autonomous mode every time we wanted to toggle brake mode on the robot when testing.

Reply:
this is true